skglm/solvers/fista.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
+from scipy.sparse import issparse
+from numba import njit
+from skglm.solvers.base import BaseSolver
+from skglm.solvers.common import construct_grad, construct_grad_sparse
+
+
+@njit
+def _prox_vec(w, z, penalty, lipschitz):
+    n_features = w.shape[0]
+    for j in range(n_features):
+        w[j] = penalty.prox_1d(z[j], 1 / lipschitz, j)
+    return w
+
+
+class FISTA(BaseSolver):
```

Comment:
What about fitting an intercept?

Reply:
I haven't really understood how to do it. I know @Klopfe is writing a piece of documentation about it, which I'll read eagerly. For now, I leave it as it and I open #97 for a subsequent PR.